ros/python/py_pubsub/py_pubsub/object_detector.py

```diff
@@ -8,6 +8,7 @@
 from rclpy.qos import qos_profile_sensor_data
```

Comment:
Meta-comment: we should think about creating different packages for the different nodes / grouped collections of nodes.